swagger.yaml

```diff
@@ -10,27 +10,461 @@ basePath: "/v1"
+
+  Bulletin:
+    type: object
+    properties:
+      id:
+        $ref: '#/definitions/id'
+      contacts:
+        type: array
+        items:
+          $ref: '#/definitions/Contact'
+      description:
+        $ref: '#/definitions/summary'
+      keywords:
+        type: array
+        description: List of keywords relating to this bulletin
+        items:
+          type: string
+        example: [ "work" , "jobless" ]
+      license:
+        description: "The standard Government license right text for the bulletin"
+        type: string
+      national_statistic:
+        $ref: '#/definitions/nationalStatistic'
+      next_release:
+        $ref: '#/definitions/nextRelease'
+      related_links:
+        type: array
+        items:
+          $ref: '#/definitions/RelatedLink'
+      related_data:
+        type: array
+        items:
+          $ref: '#/definitions/RelatedLink'
+      related_documents:
+        type: array
+        items:
+          $ref: '#/definitions/RelatedLink'
+      related_methodologies:
+        type: array
+        items:
+          $ref: '#/definitions/RelatedLink'
+      related_methodology_articles:
+        type: array
+        items:
+          $ref: '#/definitions/RelatedLink'
+      related_topics:
+        type: array
+        items:
+          $ref: '#/definitions/RelatedLink'
+      title:
+        $ref: '#/definitions/title'
+      links:
+        type: object
+        properties:
+          self:
+            $ref: '#/definitions/Link'
+          editions:
+            $ref: '#/definitions/Link'
+          latest_edition:
+            $ref: '#/definitions/Link'
+          versions:
+            $ref: '#/definitions/Link'
+          latest_version:
+            $ref: '#/definitions/Link'
+
+  EditionsList:
+    type: object
+    properties:
+      id:
+        $ref: '#/definitions/id'
+      count:
+        $ref: '#/definitions/itemCount'
+      items:
+        type: array
+        items:
+          $ref: '#/definitions/Edition'
+
+  Edition:
+    type: object
+    properties:
+      id:
+        $ref: '#/definitions/id'
+      edition:
+        $ref: '#/definitions/edition'
+      links:
+        type: object
+        properties:
+          self:
+            $ref: '#/definitions/Link'
+          bulletin:
+            $ref: '#/definitions/Link'
+          versions:
+            $ref: '#/definitions/Link'
+          latest_version:
+            $ref: '#/definitions/Link'
+
+
+  VersionsList:
+    type: object
+    properties:
+      id:
+        $ref: '#/definitions/id'
+      edition:
+        $ref: '#/definitions/edition'
+      count:
+        $ref: '#/definitions/itemCount'
+      items:
+        type: array
+        items:
+          $ref: '#/definitions/Version'
+
+  Version:
+    type: object
+    properties:
+      id:
+        $ref: '#/definitions/id'
+      edition:
+        $ref: '#/definitions/edition'
+      version:
+        $ref: '#/definitions/version'
+      alerts:
```

Comment:
`alerts` and `correction_notice` can be combined and we can use the `type` field to say whether the alert is a `correction` or `notice`.